html_css/intermediate_css/selectors.md

```diff
@@ -94,7 +94,7 @@ As always don't forget to check the [docs](https://developer.mozilla.org/en-US/d
 ## Dynamic and User Action Pseudo-classes
 These types of useful pseudo-classes can make your page feel much more dynamic and interactive.
 
-[:`focus`](https://css-tricks.com/almanac/selectors/f/focus/)
+[:`focus`](https://css-tricks.com/almanac/selectors/f/focus/) applies to an element that is currently selected by the user either through selecting it with their cursor or using their keyboard. 
```

Comment:
We should put the colon inside the code backticks here. If you could also update line 101 (the `:active` selector) that would be perfect, too.

```suggestion
[`:focus`](https://css-tricks.com/almanac/selectors/f/focus/) applies to an element that is currently selected by the user either through selecting it with their cursor or using their keyboard. 
```